tests/unit_tests/actsvg/actsvg_detector.cpp

```diff
@@ -0,0 +1,80 @@
+
+#include <array>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <type_traits>
+#include <vector>
+
+// Project include(s)
+#include "detray/core/detector.hpp"
+#include "detray/definitions/indexing.hpp"
+#include "detray/definitions/qualifiers.hpp"
+#include "detray/detectors/create_toy_geometry.hpp"
+#include "detray/geometry/surface.hpp"
+#include "detray/io/common/detail/file_handle.hpp"
+#include "detray/io/common/detector_writer.hpp"
+#include "detray/masks/cylinder2D.hpp"
+#include "detray/masks/masks.hpp"
+#include "detray/plugins/actsvg/surface_svg_converter.hpp"
+#include "detray/tracks/tracks.hpp"
+
+// Vecmem include(s)
+#include <vecmem/memory/host_memory_resource.hpp>
+
+// Actsvg include(s)
+#include "actsvg/core.hpp"
+#include "actsvg/core/defs.hpp"
+#include "actsvg/display/geometry.hpp"
+#include "actsvg/meta.hpp"
+#include "actsvg/proto/surface.hpp"
+
+using namespace actsvg;
+
+int main(int argc, char* argv[]) {
+
+    using toy_detector_t = detray::detector<detray::toy_metadata<>>;
+
+    vecmem::host_memory_resource host_mr;
+    const toy_detector_t det = detray::create_toy_geometry(host_mr, 4, 3);
+    toy_detector_t::geometry_context context{};
+
+    views::x_y x_y_view;
+
+    // Draw x-y-axis.
+    style::stroke stroke_black = style::stroke();
+    auto x_y_a =
+        draw::x_y_axes("xy", {-250, 250}, {-250, 250}, stroke_black, "x", "y");
+
+    // Create SVG file.
+    svg::file file;
+
+    file.add_object(x_y_a);
+
+    auto indices = {13, 20, 100, 150, 200, 250};
+
+    for (const auto& pair :
+         detray::views::pick(det.surface_lookup(), indices)) {
+        const auto index = pair.first;
+        const auto description = pair.second;
```

Comment:
You might be able to do this more concisely with "structured binding" :
```suggestion
    for (const auto [index, description] :
         detray::views::pick(det.surface_lookup(), indices)) {
```

Reply:
Yeah, I forgot that this returns two values, sorry